main.go

```diff
@@ -0,0 +1,18 @@
+// Package main is the entry point for the application.
+package main
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/ubuntu/ubuntu-proxy-manager/internal/app"
+)
+
+func main() {
+	app, err := app.New()
+	if err != nil {
+		log.Fatalf("failed to create app: %v", err)
```

Comment:
I would rather log.Error() + os.Exit(1) here for the sake of consistency when logging.

Reply:
hum, this is not applied when looking at the new main.go?